javaslang/src/test/java/javaslang/collection/IntMap.java

```diff
@@ -292,6 +292,11 @@ public long estimateSize() {
             public int characteristics() {
                 return spliterator.characteristics();
             }
+
+            @Override
+            public Comparator<? super T> getComparator() {
+                return null;
+            }
```

Comment:
Yes, there is. In Traversable we are using a spliterator based on a given iterator which is IteratorSpliterator. It has such implementation of ```getComparator()```:
``` 
if (hasCharacteristics(Spliterator.SORTED))
                return null;
            throw new IllegalStateException();
```
For some reason I thought that it is ok to return null. But know I understood that only Linked Maps are sorted that why I have to implement the same logic. And I will :)